applications/blogs/backend/src/main/java/pl/ds/kyanite/blogs/components/models/BlogPostTableOfContentsComponent.java

```diff
@@ -65,7 +74,10 @@ public class BlogPostTableOfContentsComponent {
   private String backToTopButtonLabel;
 
   @Getter
-  private List<BlogPostTableOfContentsItemComponent> items;
+  private List<BlogPostTableOfContentsItemComponent> subTitles;
+
+  @Getter
```

Comment:
I cannot found any of the usages of this field. Is it used anywhere?

Reply:
The new concept is to form ToC recursively. ToC itself and each subitem in its hierarchy have `subTitles` field representing child elements.

<img width="858" alt="image" src="https://github.com/websight-io/kyanite/assets/13979207/20144a07-69e9-4d18-b5b4-86a3e72a9f6b">

<img width="801" alt="image" src="https://github.com/websight-io/kyanite/assets/13979207/05d1e902-1f93-41d9-b266-249b9dbfaec5">

<img width="601" alt="image" src="https://github.com/websight-io/kyanite/assets/13979207/5ac6b0e3-2a8e-4b97-bf5c-97dd8c03bead">

<img width="514" alt="image" src="https://github.com/websight-io/kyanite/assets/13979207/31e28000-9af0-4ad2-828e-ac1a98888e04">

Reply:
I was asking about `hierarchyErrorMessage`. Sorry, assigning the question to the @Getter was misleading.

Reply:
I included this message into the template in one of the recent commits. At the moment of your comment it wasn't used, but I was leaving this option for the future.